shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-core-spring/shardingsphere-jdbc-core-spring-boot-starter/src/main/java/org/apache/shardingsphere/spring/boot/ShardingSphereAutoConfiguration.java

```diff
@@ -63,6 +64,7 @@ public class ShardingSphereAutoConfiguration implements EnvironmentAware {
     
     private String databaseName;
     
+    @Lazy
```

Comment:
Of course, in the long run, other classes should not continue to be injected into the classes registered in the Spring container in advance. But this is not easy to handle, and it is likely to require the existence of unit tests for Spring Boot 2.x in the project, and the project is currently Using Spring Boot 1.x for unit testing.